src/Components/Facility/ConsultationForm.tsx

```diff
@@ -475,6 +646,53 @@ export const ConsultationForm = (props: any) => {
     dispatch({ type: "set_form", form });
   };
 
+  const handleLinesChange = (e: any) => {
+    const form = { ...state.form };
+    const { value } = e?.target;
+    form.lines = value;
+    form.hasLines = form.lines.length > 0;
+    form.otherLines = !!form.lines.filter((i: number) => i === 7).length;
+    dispatch({ type: "set_form", form });
+  };
+
+  const handleLinesDateChange = (id: any, date: any) => {
+    const form = { ...state.form };
+    if (moment(date).isValid()) {
+      form.lines_insertion_date[id] = date;
+    }
+    dispatch({ type: "set_form", form });
+  };
+
+  const handleLinesSiteChange = (e: any, id: any) => {
+    const form = { ...state.form };
+    form.lines_site_level_fixation[id] = e.target.value;
+    dispatch({ type: "set_form", form });
+  };
+
+  const createLinesPayload = () => {
+    const payload = state.form.lines.map((id: any) => {
+      const type = lineIdToString(id);
+      const other_type = id === 7 ? state.form.other_lines : "";
+      const start_date = state.form.lines_insertion_date[id];
+      const site = state.form.lines_site_level_fixation[id];
+      return {
+        start_date: lineRequired ? start_date : undefined,
+        type: lineRequired ? type : undefined,
+        site: lineRequired ? site : undefined,
+        //other_type: lineRequired ? other_type : undefined,
```

Comment:
@Apurva-Nagar can you look into it? Backend was not accepting this param, i commented and now its working